lib/fluent/config/types.rb

```diff
@@ -74,6 +74,9 @@ def self.regexp_value(str)
       return nil unless str
       return Regexp.compile(str) unless str.start_with?("/")
       right_slash_position = str.rindex("/")
+      if right_slash_position < str.size - 3
+        raise ConfigError, "invalid regexp: missing right slash: #{str}"
```

Comment:
Could you add `Fluent::`?
We want to use `Fluent::ConfigError` for new code.

Reply:
Just added.